homeassistant/components/romy/strings.json

```diff
@@ -0,0 +1,52 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "unlock_http_interface": {
+        "title": "Local Http Interface is locked",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "password": "Please provide password (8 characters, see QR Code under the dustbin)."
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to set up {name} at {host}?",
+        "title": "Discovered romy"
```

Comment:
Perhaps we want to be consistent with `ROMY` or `ROMY Vacuum Cleaner` here?

Reply:
ups thx, changed to ROMY Vacuum Cleaner

Reply:
or not, brand name is not allowed (commit sanity check complained) removed title now